test/libgit2-online.jl

```diff
@@ -22,13 +22,32 @@ mktempdir() do dir
             try
                 repo_path = joinpath(dir, "Example2")
                 # credentials are required because github tries to authenticate on unknown repo
-                cred = LibGit2.UserPasswordCredentials("","") # empty credentials cause authentication error
+                cred = LibGit2.UserPasswordCredentials("JeffBezanson", "hunter2") # make sure Jeff is using a good password :)
```

Comment:
Hey, how did you get my password?!! :)
Doesn't matter to me, I just want AV to work again.